core-sample-app/src/main/AndroidManifest.xml

```diff
@@ -90,6 +90,11 @@
             android:configChanges="orientation|screenSize|keyboardHidden|smallestScreenSize|screenLayout"
             android:hardwareAccelerated="true"
             android:label="@string/fullscreen_example" />
+        <activity
```

Comment:
No need to create a new sample activity. Can we reuse the existing full screen sample activity?

Reply:
Noted 👍

Reply:
Changes are made as per suggestion, removed `FullscreenOnOrientationExampleActivity` and update `FullscreenExampleActivity` accordingly.